api/core/model_runtime/model_providers/tongyi/llm/qwen-max-longcontext.yaml

```diff
@@ -8,54 +8,68 @@ model_properties:
 parameter_rules:
   - name: temperature
     use_template: temperature
-    default: 1.0
-    min: 0.0
+    type: float
+    default: 0.85
+    min: 0.01
     max: 2.0
     help:
       zh_Hans: 用于控制随机性和多样性的程度。具体来说，temperature值控制了生成文本时对每个候选词的概率分布进行平滑的程度。较高的temperature值会降低概率分布的峰值，使得更多的低概率词被选择，生成结果更加多样化；而较低的temperature值则会增强概率分布的峰值，使得高概率词更容易被选择，生成结果更加确定。
       en_US: Used to control the degree of randomness and diversity. Specifically, the temperature value controls the degree to which the probability distribution of each candidate word is smoothed when generating text. A higher temperature value will reduce the peak value of the probability distribution, allowing more low-probability words to be selected, and the generated results will be more diverse; while a lower temperature value will enhance the peak value of the probability distribution, making it easier for high-probability words to be selected. , the generated results are more certain.
+  - name: max_tokens
+    use_template: max_tokens
+    required: false
```

Comment:
It's quite confused when setting all these common parameters (max_tokens, top_p and etc.) into not required by default.
These configs are not just for calling the LLM APIs, but also for the balance of productive and deliverable app constructions.

Reply:
What I hope is that when the front-end is displayed, these options can be unchecked by default.

You are more familiar with the project, so I will follow your suggestion.

Reply:
LGTM.

Reply:
okay, done